app/uk/gov/hmrc/excisemovementcontrolsystemapi/connectors/EISSubmissionConnector.scala

```diff
@@ -76,12 +78,33 @@ class EISSubmissionConnector @Inject()
       }
   }
 
+  private def wrapXmlInControlDocument(request: ValidatedXmlRequest[_]): NodeSeq = {
```

Comment:
I suspect this will be a reusable thing, if so this may be moved to its own file and object/class. 

Also I wonder if may be better to pass the messageIdentifier and the body as parameter instead of the ValidateXmlRequest as this may suggest the function may be doing some sort of validation with that. Just a thought.

Reply:
I don't think this will be used outside of the submission connector so might as well just keep it here.

I have changed the params to the method though for the messageIdentifier and innerXml body